adafruit_wiznet5k/adafruit_wiznet5k_dns.py

```diff
@@ -275,7 +273,7 @@ def gethostbyname(self, hostname: bytes) -> Union[int, bytes]:
                     return -1
                 time.sleep(0.05)
             # recv packet into buf
-            buffer = self._sock.recv(512)  # > UDP payload length
+            buffer = self._sock.recv()
```

Comment:
Why are we removing the 512-byte UDP payload length being set here? Is there no longer a limit for how much we want to recv?

Reply:
I have no idea TBH. I'll revert it to `recv(512)` to match the current `main` branch. Great catch. Thank you!

Reply:
thx!